modules/node-shared/src/test/scala/org/tessellation/node/shared/domain/seedlist/RangeSuite.scala

```diff
@@ -0,0 +1,79 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.Range.RangeValidationResult
+
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.NonNegLong
+import weaver.FunSuite
+import weaver.scalacheck.Checkers
+
+object RangeSuite extends FunSuite with Checkers {
+
+  implicit class DecodeOps(input: String) {
+    def asRange: Option[Range] = Range.decode(input).toOption
+    def asRangeValidation: RangeValidationResult[Range] = Range.decode(input)
```

Comment:
it does nothing additional so I see no point in making an implicit `asRangeValidation` instead of using just `Range.decode()`